src/quo2/components/notifications/activity_log/style.cljs

```diff
@@ -6,7 +6,7 @@
    :flex-grow          1
    :align-items        :flex-start
    :padding-top        8
-   :padding-horizontal 12
+  ;;  :padding-horizontal 12
```

Comment:
We can just remove it)

Reply:
Sorry for that, I pushed the changes, but not updated them here due to the current service incident.